Client/Application/WebServer.swift

```diff
@@ -18,8 +18,10 @@ class WebServer {
     }
 
     func start() -> Bool {
-        if !server.running {
-            try! server.startWithOptions([GCDWebServerOption_Port: 6571, GCDWebServerOption_BindToLocalhost: true, GCDWebServerOption_AutomaticallySuspendInBackground: true])
+        withExtendedLifetime(server) {
```

Comment:
I'm also worried about the `try!` there. Can we also wrap that in a `do {} catch {}` and log if it raises an error? With the current code, obviously auto-converted, we would hard crash during startup. We can make it more robust and still run with a broken internal web server. (Which will lead to better bug reports than a possible startup crash)